mgmtsystem/openstack_infra.py

```diff
@@ -7,6 +9,9 @@
 from base import MgmtSystemAPIBase
 
 
+Node = namedtuple('Node', ['uuid', 'name', 'power_state', 'provision_state'])
```

Comment:
We might provide some unified interface and unified object for hosts and such. Strings are no fun.

I had to add a separate method for nodes to not break the interface.